tests-legacy/Unit/Adapter/Module/Tab/ModuleTabRegisterTest.php

```diff
@@ -192,7 +192,7 @@ protected function buildFilesystemMock()
 
     protected function buildRoutingConfigLoaderMock()
     {
-        $moduleRoutingLoader = $this->getMockBuilder(LoaderInterface::class)
+        $moduleRoutingLoader = $this->getMockBuilder(Loader::class)
```

Comment:
We should aim for interface when possible. Is this possible here ?

Reply:
We are using methods not declared in the interface, here, we depend on the Loader type, not the interface.